apps/style/applab/style.scss

```diff
@@ -96,23 +96,24 @@ div#visualizationResizeBar {
   }
   div#visualizationResizeBar {
     left: $width;
-    line-height: $width;
+    line-height: $height;
   }
   html[dir='rtl'] div#visualizationResizeBar {
     right: $width;
   }
   div#visualization.responsive > * {
-    -webkit-transform: scale(0.875);
-    -ms-transform: scale(0.875);
-    transform: scale(0.875);
+    -webkit-transform: scale(1.09375);
+    -ms-transform: scale(1.09375);
+    transform: scale(1.09375);
```

Comment:
These are scale factors `[1.0, 0.875, 0.75, 0.625, 0.5]` times another scale factor `new_app_height / original_app_height = 450 / 400 = 1.25`. I will try to refactor these into products of two factors so that it is a bit more clear where the numbers are coming from.